pkg/client/group_mgmt_client.go

```diff
@@ -411,9 +330,12 @@ func processResponse(client *GroupMgmtClient, response *resty.Response, path str
 				return nil, err
 			}
 		}
-		// process success response
-
-		return unwrapData(response.Body(), respHolder)
+		// unmarshall response data
+		dataIntf, totalRows, err := unwrapData(response.Body(), respHolder)
+		if params != nil && params.Page != nil && totalRows != nil {
+			params.Page.TotalRows = totalRows
```

Comment:
This comment isn't related to this code review, but it is a convenient place for me to mention it anyway.  :)

The pagination.go file is in a param/pagination package.  Is there a reason not to put it in the param package directly?

Reply:
We need to initialze the totalRow which is not visible outside this pkg

Reply:
I probably wasn't clear.  I'm suggesting moving param/pagination/pagination.go to param/pagination.go, unless there is a good reason to have a separate package for it.